staging/src/k8s.io/apiserver/pkg/admission/exclusion/exclusion.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package exclusion
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// Included is the set of resources that the expression-based admission controllers
+// should intercept.
+// If a resource is transient, i.e., not persisted in the storage, the resource must be
+// in either Included or Excluded list.
+var Included = sets.New[schema.GroupVersionResource](
+	schema.GroupVersionResource{Group: "", Version: "v1", Resource: "bindings"},
+	schema.GroupVersionResource{Group: "", Version: "v1", Resource: "pods/attach"},
+	schema.GroupVersionResource{Group: "", Version: "v1", Resource: "pods/binding"},
+	schema.GroupVersionResource{Group: "", Version: "v1", Resource: "pods/eviction"},
+	schema.GroupVersionResource{Group: "", Version: "v1", Resource: "pods/exec"},
+	schema.GroupVersionResource{Group: "", Version: "v1", Resource: "pods/portforward"},
+
+	// ref: https://github.com/kubernetes/kubernetes/issues/122205#issuecomment-1927390823
+	schema.GroupVersionResource{Group: "", Version: "v1", Resource: "serviceaccounts/token"},
+)
+
+// Excluded is the set of resources that the expression-based admission controllers
+// should ignore.
+var Excluded = sets.New[schema.GroupVersionResource](
```

Comment:
I think we can just do group/resource and treat all versions the same

Reply:
My rationale for having versions here is that, if we want to revise a decision we can put different versions into different bucket. For example, in a `v1alpha1` version we exclude some resource but later we want to include it in `v1beta`, we can include `v1beta1`, leave `v1alpha1` as-is for a smoother transition. 

Would you agree? Thank you.

Reply:
I think treating versions differently is going to lead to accidents (where we don't consistently exclude all versions of a resource) and really weird behavior with `matchPolicy: Equivalent`

Reply:
alright. good point. there is almost never an use case for having different versions into different categories. I am now changing to use schema.GroupResource instead.